### ISSUE TITLE
Compatibility with linter 2.3.0

### ISSUE THREAD

**Issue by soffes** [leonelgalan/linter-markdownlint#10]
I couldn't get this to work with linter 2.3.0. The following works for me though:

```
apm install linter@2.2.0
apm install linter-ui-default@1.7.1
apm install intentions@1.1.5
apm install busy-signal@1.4.3
apm install linter-markdownlint@1.0.2
gem install mdl --version 0.5.0
```

I noticed `busy-signal` also isn't up to date with this. I didn't try with just downgrading `linter` and not `busy-signal`.

**Comment by leonelgalan** [comment]
[Linter 2.3.0 removed support for legacy linter APIs](https://github.com/steelbrain/linter/blob/master/CHANGELOG.md#230): I'll start the update.

**Comment by leonelgalan** [comment]
@soffes Thanks for your feedback! I just released 2.0.0. Which supports the newer Linter API!
